### DISCUSSION TITLE
検索結果の記事に付けられたタグが綺麗に表示されない

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#386] [要望・アイデア・不具合]
ウィンドウ幅が狭い場合、検索結果の記事に付けられたタグの改行が奇妙になったり見切れたりすることがあります。

この問題はスマホで顕著に見られます。

![presentation][1]|![chocolate][2]
:--:|:--:
プレゼンテーションが一文字毎に改行されている|最後のタグが見切れている（最後のタグは `chocolatey`）

[1]: https://user-images.githubusercontent.com/75878492/208601889-e15d106b-0ce4-4a35-a293-3e2aca3bf0dd.jpg
[2]: https://user-images.githubusercontent.com/75878492/208601893-3168046d-711d-4636-b9a2-3f94173de964.jpg

> 左の例：
> https://qiita.com/search?sort=created&q=tag%3Amarp+tag%3Amarkdown+tag%3A%E3%83%97%E3%83%AC%E3%82%BC%E3%83%B3%E3%83%86%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3+tag%3Agithub-pages+tag%3Agithubactions
>
> 右の例：
> https://qiita.com/search?sort=created&q=tag%3Alatex+tag%3Alualatex+tag%3Atexlive+tag%3Avscode+tag%3Achocolatey

検索結果内のタグの羅列は、全体として 1 行に収まるように設定されているようですが、タグ内で改行されるよりタグが複数行に渡って表示される方が綺麗に見えると思います。

トレンドや特定のタグのページでは複数行に渡って表示されるようになっています。

<img src="https://user-images.githubusercontent.com/75878492/208605600-071ac871-330f-4aee-aadd-af95c18ef3c1.jpg" width="50%" alt="初心者タグ">

> https://qiita.com/tags/%e5%88%9d%e5%bf%83%e8%80%85

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
発見していただき、ありがとうございます🙇
こちら修正いたします！

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
お待たせしました🙇こちら修正が完了いたしました！

**Reply by yarakos95** [reply]
対応ありがとうございます！！
問題が解決していることを確認しました 😄
